packages/embark/src/lib/modules/console/suggestions.ts

```diff
@@ -63,7 +63,11 @@ export default class Suggestions {
   }
 
   public getSuggestions(cmd: string, cb: (results: SuggestionsList) => any) {
+    // Don't bother returning suggestions for empty commands or for
+    // commands that already have `(` or `)` to avoid executing code
     if (cmd === "") { return cb([]); }
+    if (cmd.indexOf("(") !== -1 || cmd.indexOf(")") !== -1) { return cb([]); }
```

Comment:
Could we make this be an `and` so that we only stop suggesting once they've closed the parenthesis?
```suggestion
    if (cmd.indexOf("(") !== -1 && cmd.indexOf(")") !== -1) { return cb([]); }
```

Reply:
Well, as soon as there is one parenthesis, the eval will crash

Reply:
True, I guess either way would work then.